services/model_service.py

```
@@ -307,36 +320,65 @@ def _try_aria2c(
+
+            if destination.exists() and destination.stat().st_size > 0:
+                logger.info(f"✅ Download complete with hf_hub_download: {destination}")
                 return True
-            else:
-                logger.warning(f"hf-transfer failed with exit code {result.returncode}")
-                return False
+
+            logger.warning("hf_hub_download returned but file missing or empty")
+            return False
 
         except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)